database/oracle/README.md

```diff
@@ -0,0 +1,80 @@
+# oracle
+The golang oracle driver [godror](https://github.com/godror/godror) does not natively support executing multiple statements in a single query. 
+Here are the strategies for splitting the migration text into separately-executed statements:
+1. If there is no PL/SQL statement in a migration file, the `semicolons` will be the separator
+1. If there is any PL/SQL statement in a migration file, the separator will be `---` in a single line or specified by `x-plsql-line-separator`, 
+   And in this case the multiple statements cannot be used when a statement in the migration contains a string with the given `line separator`.
+
+`oracle://user:password@host:port/sid?query`
+
+| URL Query  | WithInstance Config | Description |
+|------------|---------------------|-------------|
+| `x-migrations-table` | `MigrationsTable` | Name of the migrations table in UPPER case |
+| `x-plsql-line-separator` | `PLSQLStatementSeparator` | a single line which use as the token to spilt multiple statements in single migration file (See note above), default `---` |
+
+## Runtime dependency
+
+Although an Oracle client is NOT required for compiling, it is at run time. One can download it from https://www.oracle.com/database/technologies/instant-client/downloads.html
+
+## Supported & tested version
+- 12c-ee
+- 18c-xe
+
+## How to use
+
+In order to compile & run the migration against Oracle database, basically it will require:
+
+## Build cli
+
+```bash
+$ cd /path/to/repo/dir
+$ go build  -tags 'oracle' -o bin/migrate github.com/golang-migrate/migrate/v4/cli
+```
+
+## Configure Oracle database
+1. Example Oracle version: `Oracle Database Express Edition`, check [here](https://docs.oracle.com/cd/B28359_01/license.111/b28287/editions.htm#DBLIC119) from version details.
```

Comment:
In addition to the license info, link to the download. I _think_ this is what you used: https://www.oracle.com/database/technologies/xe-downloads.html